rootfs/usr/bin/playtronos-systemreport

```diff
@@ -231,9 +250,9 @@ fi
 echo ""
 echo -e "${CYAN}Building report archive${ENDCOLOR}"
 cd "${REPORT_DIR_BASE}" && tar cvfz "${REPORT_NAME}.tar.gz" "${REPORT_NAME}"
-mv "${REPORT_DIR}.tar.gz" "/tmp/${REPORT_NAME}.tar.gz"
-chown ${TARGET_USER}: "/tmp/${REPORT_NAME}.tar.gz"
+mv "${REPORT_DIR}.tar.gz" $OUTPUT_PATH
```

Comment:
Need quotes around `$OUTPUT_PATH` to be able to handle paths with spaces in them. Same for the line below.